src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,140 @@
+package duke;
+
+import tasks.Deadline;
+import tasks.TaskList;
+import tasks.Task;
+import tasks.Todo;
+import tasks.Event;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Storage class to handle saving and writing data
+ */
+public class Storage {
```

Comment:
Complete and perfect saving and loading task list in this class. I think it is good!